internal/exec/stages/disks/luks.go

```diff
@@ -531,3 +570,149 @@ func (d LuksDump) hasFlag(flag string) bool {
 	}
 	return false
 }
+
+// collect the cex card domains xx.xxxx
+func getAllCexOnlineAPQNs() (string, error) {
+	apDir := "/sys/bus/ap/devices/"
+	uEvent := "/uevent"
+	var ret []string
+	// cards details are in form of link file points to card directory.
+	event, _ := os.ReadDir(apDir)
+	if len(event) == 0 {
+		return "", fmt.Errorf("adjunct processor not found")
+	}
+	// match the regexp for pattern xx.xxxx
+	for _, dir := range event {
+		match := cexRegx.FindStringSubmatch(dir.Name())
```

Comment:
Since we don't have any submatches in our pattern, we could just use `MatchString()` instead here. That'll avoid the awkward `strings.Join()` below.

Reply:
Will correct with string concatenate.